.github/workflows/github.yml

```diff
@@ -384,3 +384,28 @@ jobs:
         name: ${{ env.VCMI_PACKAGE_FILE_NAME }}
         path: |
           ${{ env.ANDROID_APK_PATH }}
+
+
+  deploy-src:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: pwsh
+    steps:
+        - uses: actions/checkout@v4
+          with:
+            submodules: recursive
+            
+        - name: Create source code archive (including submodules)
+          run: |
+            git archive HEAD -o "release.zip" --worktree-attributes -v
+            git submodule update --init --recursive
+            git submodule --quiet foreach 'cd "$toplevel"; zip -ru "release.zip" "$sm_path"'
+            unzip -q release.zip -d ./src
+            
+        - name: Upload source code archive
+          uses: actions/upload-artifact@v4
+          with:
+            name: src
```

Comment:
Can we use same naming scheme as other artifacts?, e.g.:
`9040593311-vcmi-PR-3933-844da66 - ios.zip`
Currently sources are named simply `src.zip`

Reply:
I've changed the naming scheme to `${{ env.VCMI_PACKAGE_FILE_NAME }}`, would this be ok?